release-team/role-handbooks/enhancements/README.md

```diff
@@ -327,7 +339,7 @@ Here's where this enhancement currently stands:
 - [ ] KEP readme using the [latest template](https://github.com/kubernetes/enhancements/tree/master/keps/NNNN-kep-template) has been merged into the k/enhancements repo.
 - [ ] KEP status is marked as `implementable` for `latest-milestone: { CURRENT_RELEASE }`. KEPs targeting `stable` will need to be marked as `implemented` after code PRs are merged and the feature gates are removed.
```

Comment:
@sreeram-venkitesh, could you add this change as well? Thanks!

During the enhancements freeze outreach, marking a KEP **_targeting_** stable as `implemented` is incorrect.

(Consider the scenario where a KEP slips to the next cycle, but kep.yaml already says `implemented`, which is misleading.)

```suggestion
- [ ] KEP status is marked as `implementable` for `latest-milestone: { CURRENT_RELEASE }`.
```

---

Should we move this line to the code freeze outreach template?

```
KEPs targeting `stable` need to be marked as `implemented` after code PRs are merged and the feature gates are removed.
```

If all PRs for stable graduation are merged, it makes sense at that point to mark the KEP as `implemented`.

However, a note: we won't push a KEP to raise a code freeze exception request if, on code freeze day, the only thing missing is marking the KEP as `implemented` in kep.yaml.

Reply:
I agree, I was talking with @ArkaSaha30 yesterday about how it only makes sense to update the status as implemented after the code freeze. This is also an item in week 0 tasks, to contact KEPs from the previous milestone to make sure stable KEPs have this updated properly. I'll make this change.

Reply:
I've made this change, but I have a question about not pushing folks to raise code freeze exception. If a KEP has all its code PRs merged, we can mark it as tracked, even if its not marked as implemented right? Since this change can be made to k/enhancements even after the code freeze. So updating the status to `implemented` is not a criteria for passing code freeze, its just that we will remind folks to do so with the code freeze template. I think then it is better to have this in the message where we mark the KEP as tracked for code freeze. Since KEPs which don't make the code freeze cut cannot be marked as `implemented`.

Reply:
Please check https://github.com/kubernetes/sig-release/pull/2518/commits/38ca7c7bf71031d9e9c577abfca4ddfd53348ed0.